topics/reference_guide/custom_language_support/navbar.md

```diff
@@ -0,0 +1,45 @@
+[//]: # ("title: Navigation Bar")
```

Comment:
Please add an entry in `intellij-sdk-docs/ijs.tree` file, otherwise, it won't be listed in the page structure.